stetho-sample/src/debug/java/com/facebook/stetho/sample/APODContentProviderDumper.java

```diff
@@ -0,0 +1,112 @@
+package com.facebook.stetho.sample;
+
+import android.content.ContentResolver;
+import android.database.Cursor;
+import com.facebook.stetho.dumpapp.DumpException;
+import com.facebook.stetho.dumpapp.DumperContext;
+import com.facebook.stetho.dumpapp.DumperPlugin;
+
+import java.io.PrintStream;
+import java.util.List;
+
+public class APODContentProviderDumper implements DumperPlugin {
+
+  private static final String NAME = "content";
+
+  private static final String CMD_LIST = "list";
+  private static final String USAGE_LIST = CMD_LIST;
```

Comment:
The primary reason is so that if/when you add arguments like `USAGE_REMOVE_ROW` you dont need to create a new one and modify it below.